plugins/spark_upgrade/execute_piranha.py

```diff
@@ -0,0 +1,117 @@
+#  Copyright (c) 2023 Uber Technologies, Inc.
```

Comment:
we might want to move it outside in plugins/base so all plugins can use this

Reply:
Since forever, we have maintained one LICENSE file for entire repo and such license comments for each file. :|